sync/clients/sync.py

```diff
@@ -119,6 +119,31 @@ def create_project_submission(self, project_id: str, submission: dict) -> dict:
             )
         )
 
+    def create_project_recommendation(self, project_id: str, **kwargs) -> dict:
```

Comment:
`kwargs` is a very generic name. Maybe rec_params is better unless we want to be more explicit about the options for creating a rec - optoinal parmas in place of the catch-all ** param